client/rest-high-level/src/test/java/org/elasticsearch/client/RestHighLevelClientTests.java

```diff
@@ -107,6 +107,9 @@
 
 public class RestHighLevelClientTests extends ESTestCase {
```

Comment:
we don't run this test for subclients like tasks() migraiton() etc
maybe we should..

Reply:
Oh! We almost certainly should. But we'd have to do some "interesting" work to get the specs in place. We probably could come up with something though.